i/logo.svg

```diff
@@ -0,0 +1,51 @@
+<svg
+  class="crm-logo"
+  aria-label="[title + description]"
+  role="img"
+  version="1.1"
+  viewBox="0 0 229.76 59.783"
+  xml:space="preserve"
+  xmlns="http://www.w3.org/2000/svg"
+  >
+  <title>{ts}CiviCRM Logo{/ts}</title>
+  <description>{ts}The word CiviCRM with an intersecting blue and green triangle{/ts}</description>
+  <style type="text/css">
+    .crm-logo-text {
+      fill: var(--crm-c-logo-text, #000);
+    }
+     @media (prefers-color-scheme: dark) {
+       .crm-logo-text {
+         fill: var(--crm-c-logo-text, #fff);
+       }
+     }
+
+    .crm-logo-triangle-1 {
+      fill: var(--crm-c-logo-triangle-1, #086287);
+    }
+    .crm-logo-triangle-2 {
+      fill: var(--crm-c-logo-triangle-2, #81C459);
```

Comment:
Just to note: these css vars aren't actually functional when the svg is used with `<img>` like it is here, because styles dont penetrate into the `<img>`. But they aren't doing any harm; it may be supported in future; and if you really wanted to you could `file_get_contents('i/logo.svg')` and then inline in the html and use it